sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -71,8 +71,7 @@ import org.apache.spark.util.ArrayImplicits._
   since = "1.5.0",
   group = "string_funcs")
 // scalastyle:on line.size.limit
-case class ConcatWs(children: Seq[Expression])
-  extends Expression with ImplicitCastInputTypes {
+case class ConcatWs(children: Seq[Expression]) extends Expression with ImplicitCastInputTypes {
```

Comment:
Can we remove those unrelated changes? those changes make it harder to backport, cherry-pick, revert, etc.

Reply:
I can do that, but all scala style tests are failing then.